src/peer_manager.rs

```diff
@@ -446,90 +446,50 @@ impl PeerManager {
     /// elapsed since the candidate was inserted.
     pub fn verify_candidate(
         &mut self,
-        new_pub_id: &PublicId,
```

Comment:
we still have many other places using this `new_pub_id`, do they need to be replaced as well?

Reply:
I replaced that one for convenience because it clashed with the other and required `new_pub_id: ref pub_id,`. I don't think we need to change the other ones soon.